examples/circuitChat.html

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <script src="//unpkg.com/circuit-sdk@latest/circuit.min.js" async></script>
+    <script type="module" src="../circuit-chat.js" defer></script>
+    <script type="module" src="../circuit-conversations-list.js" defer></script>
+    <style>
+      /* Global page styles which are inherited by element */
+      body {
+        font-family: sans-serif;
+        font-size: 13px;
+      }
+
+      input {
+        width: 150px;
+        margin: 5px 0px 10px 0px;
+      }
+
+      .conversation-button-container { 
+        padding-bottom: 20px;
+        display: flex;
+        align-items: center;
+      }
+
+      .conv-btn {
+        margin-left: 20px;
+      }
+
+      .component-container {
+        display: flex;
+      }
+
+      circuit-conversations-list::part(conversation-list) {
+        width: 700px;
+        max-height: 200px;
+        border: 1px solid green;
+        overflow-y: auto;
+      }
+
+      circuit-conversations-list::part(circuit-conversation) {
+        height: 25px;
+        display: flex;
+        align-items: center;
+        border-top: 1px solid #dadada;
+      }
+
+
+      circuit-chat {
+        padding-left: 30px;
+        /* Padding of the item content */
+        --item-padding: 0 0 0 10px;
+      }
+
+      /* Style of the send button */
+      circuit-chat::part(btn) {
+        float: right;
+        padding-left: 10px;
+        margin-top: 15px;
+      }
+
+      /* Style of the input field */
+      circuit-chat::part(inp) {
+        width: 300px;
+        margin-top: 15px;
+      }
+
+      /* Main container */
+      circuit-chat::part(main-container) {
+        width: 700px;
+      }
+
+      /* Conversation Title */
+      .converation-list-title, circuit-chat::part(conversation-title) {
+        padding-bottom: 10px;
+        font-size: 24px;
```

Comment:
Too big. In general we should not have many css definitions in the example. It should show as plain as possible, not only to make the example smaller, but also to see how the component looks by default.

For example in my call examples, the button styling is done as default in the component.